js_basics/task4.js

```diff
@@ -0,0 +1,14 @@
+// Завдання 4: Числа та булі
+
+// Створіть змінну яка буде символізувати вік певної персони.
+
+// Після чого створіть булеву змінну яка буде визначати чи особа повнолітня базуючись на заченні з попередньої змінної.
+
+// Виведіть усі змінні в консоль
+
+
+const personAge = 18;
+
+const checkAge = personAge == 18;
```

Comment:
if `personAge` equals to 19 then person is not adult? :)

Reply:
use `>=` operator

Reply:
`checkAge` is bad name for variable since this name represents the action but not some value, use `isAdult`